src/main/java/com/linecorp/armeria/client/circuitbreaker/NonBlockingCircuitBreaker.java

```diff
@@ -186,6 +181,21 @@ private void logStateTransition(CircuitState circuitState, @Nullable EventCount
         }
     }
 
+    private void notifyStateChanged(CircuitState circuitState) {
+        config.listeners().forEach(listener -> {
+            listener.onStateChanged(this, circuitState);
+            listener.onEventCountUpdated(this, EventCount.ZERO);
```

Comment:
Could we wrap each notification with `try .. catch ..` so that a listener triggering an exception does not prevent other listeners from getting notification?